src/core/qgsattributetableconfig.cpp

```diff
@@ -198,6 +198,14 @@ void QgsAttributeTableConfig::readXml( const QDomNode &node )
 
   mSortExpression = configNode.toElement().attribute( QStringLiteral( "sortExpression" ) );
   mSortOrder = static_cast<Qt::SortOrder>( configNode.toElement().attribute( QStringLiteral( "sortOrder" ) ).toInt() );
+  switch ( mSortOrder )
```

Comment:
ok, was just a way to simplify  in case to add new enum values... but it s a improbable case.